x-pack/plugins/security_solution/public/management/components/artifact_list_page/components/no_data_empty_state.test.ts

```diff
@@ -60,51 +60,85 @@ describe('When showing the Empty State in ArtifactListPage', () => {
     expect(renderResult.queryByTestId('header-page-title')).toBe(null);
   });
 
-  it('should open create flyout when primary button is clicked', async () => {
-    render();
-    const addButton = await renderResult.findByTestId('testPage-emptyState-addButton');
+  describe('and user is allowed to Create entries', () => {
+    it('should show title, about info and add button', async () => {
+      render();
 
-    act(() => {
-      userEvent.click(addButton);
+      await waitFor(() => {
```

Comment:
Could we check here that the text is the expected one?

Reply:
Yep, here is the modification: 32ebbba